api/v1alpha4/machinedeployment_types.go

```diff
@@ -57,6 +60,12 @@ type MachineDeploymentSpec struct {
 	// +kubebuilder:validation:MinLength=1
 	ClusterName string `json:"clusterName"`
 
+	// RolloutAfter performs a rollout of the MachineDeployment
+	// the first reconciliation loop where the given time is before now().
```

Comment:
I think this should be `after now()` right?

So reading this it suggests that we are just updating the machines independent of the spec? Is it worth adding a bit more context here, something like, `Any changes to the spec are not affected by this field and will be rolled out as normal`?

Reply:
>I think this should be after now() right?

If the given time is before Now() / Now() is after the given time then the given time has passed. It needs rollout.

Reply:
Time conversations are hard 😅  Comment LGTM, thanks